ui-tests/src/test/resources/features/wip/data-buckets.feature

```diff
@@ -161,3 +162,215 @@ Feature: Test functionality of data buckets during creation of integration
+    When she creates mapping from "myleadsource" to "LEAD"
+
+    And scroll "top" "right"
+    And click on the "Done" button
+
+    ##################### step step step step MAPPING step step ##################################
+
+    And She adds integration "step" on position "3"
+    And she selects "Data Mapper" integration step
+    Then she is presented with data mapper ui
+
+    Then check that data bucket "1 - SQL Result" is available and opens it
+    Then check that data bucket "2 - SQL Result" is available and opens it
+    Then check that data bucket "3 - SQL Result" is available and opens it
+    Then check that data bucket "4 - SQL Result" is available and opens it
+
+
+    When she creates mapping from "firma" to "COMPANY"
+    #change something so it is unique entry
+    When she creates mapping from "mysurname" to "MYNAME"
+    When she creates mapping from "myname" to "MYSURNAME"
+    When she creates mapping from "myleadsource" to "LEAD"
+
+    And scroll "top" "right"
+    And click on the "Done" button
+
+
+    ##################### step step step MAPPING step step ##################################
+    And She adds integration "step" on position "2"
+    And she selects "Data Mapper" integration step
+    Then she is presented with data mapper ui
+
+    Then check that data bucket "1 - SQL Result" is available and opens it
+    Then check that data bucket "2 - SQL Result" is available and opens it
+    Then check that data bucket "3 - SQL Result" is available and opens it
+
+
+    When she creates mapping from "firma" to "COMPANY"
+    When she creates mapping from "myname" to "MYNAME"
+    When she creates mapping from "mysurname" to "MYSURNAME"
+
+    And scroll "top" "right"
+    And click on the "Done" button
+
+    ##################### step step MAPPING step step step ##################################
+    And She adds integration "step" on position "1"
+    And she selects "Data Mapper" integration step
+    Then she is presented with data mapper ui
+
+    Then check that data bucket "1 - SQL Result" is available and opens it
+    Then check that data bucket "2 - SQL Result" is available and opens it
+
+
+    When she creates mapping from "firma" to "COMPANY"
+    When she creates mapping from "mysurname" to "MYSURNAME"
+
+    And scroll "top" "right"
+    And click on the "Done" button
+
+    ##################### step MAPPING step step step step ##################################
+    And She adds integration "step" on position "0"
+    And she selects "Data Mapper" integration step
+    Then she is presented with data mapper ui
+
+    Then check that data bucket "1 - SQL Result" is available and opens it
+
+    When she creates mapping from "firma" to "COMPANY"
+
+    And scroll "top" "right"
+    And click on the "Done" button
+
+
+
+    ##################### check warnings ##################################
+
+    And she checks that there is no warning inside of step number "0"
```

Comment:
Could we refactor this step a bit to take a single number && range && list of steps to check maybe? Range is probably most valuable.